### PR TITLE
add zturn lite board

### PR DESCRIPTION
**Pull request by rroohhh**
There is a version of this board with the `xc7z007s` aswell, not sure how to best handle that.

### CODE REVIEW

**Comment by whitequark** [comment]
Look at Versa ECP5.

File: nmigen_boards/zturn_lite.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+__all__ = ["ZTurnLitePlatform"]
+
+
+class ZTurnLitePlatform(Xilinx7SeriesPlatform):
+    device = "xc7z010"
+    package = "clg400"
+    speed = "1"
+
+    resources = []
```
Comment (whitequark) [inline nmigen_boards/zturn_lite.py]: I'm not sure I want to merge this as-is -- no peripherals at all?

Comment (rroohhh) [reply nmigen_boards/zturn_lite.py]: the only external peripherals are connected to the cpu, so they have to be accessed through the `ps7` block

File: nmigen_boards/zturn_lite.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+    connectors = [
+        Connector(
+            "expansion", 0, 
+            "-   -  "
+            "B19 E17"
```
Comment (whitequark) [inline nmigen_boards/zturn_lite.py]: This isn't correct. The concatenated string looks like `"... B19 E17A20 D18 ..."`

File: nmigen_boards/zturn_lite_z007s.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+            "-   -   "
+            "-   -   "
+            "-   -   "
+            "-   -   "),
+    ]
+
+
+if __name__ == "__main__":
+    from ._blinky import Blinky
+    ZTurnLiteZ007SPlatform().build(Blinky(), do_program=True)
```
Comment (whitequark) [inline nmigen_boards/zturn_lite_z007s.py]: It doesn't make any sense to run blinky on a board with no LEDs.

**Comment by whitequark** [comment]
Thanks.

**Pull request merged**